### PR TITLE
howto: add auto-complete example

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

**Comment by kenhys** [comment]
@kou Could you review this PR?

File: how-to/index.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ This document describes about useful information for specific situations.
 
 If you have useful information that isn't listed in this page, please send a pull request to [pgroonga/pgroonga.github.io](https://github.com/pgroonga/pgroonga.github.io). Please share your useful information!
 
+## How to use auto complete feature with PGroonga
```
Comment (kou) [inline how-to/index.md]: "with PGroonga" is needless.

File: how-to/autocomplete.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+--------------
+ autocomplete
+(1 rows)
+```
+
+The result contains `autocomplete` as auto complete candidate term.
+
+## Use prefix romaji katakana search against auto complete candidate readings
+
+Here is the sample data for RK search.
```
Comment (kou) [inline how-to/autocomplete.md]: Searching with romaji example is missing.

File: how-to/autocomplete.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
+---
+title: How to use auto complete feature with PGroonga
+---
+
+# How to use auto complete feature with PGroonga
+
+PGroonga can't use Groonga's suggest plugin directly, so in this
```
Comment (kou) [inline how-to/autocomplete.md]: We don't fold long lines in this site.

File: how-to/index.md (hunk 1 above)
Comment (kenhys) [inline how-to/index.md]: OK, I'll fix it.

File: how-to/autocomplete.md (hunk 3 above)
Comment (kenhys) [inline how-to/autocomplete.md]: ditto

File: how-to/autocomplete.md (hunk 2 above)
Comment (kenhys) [inline how-to/autocomplete.md]: Thanks. I've missed it.

**Comment by kenhys** [comment]
I've fixed PR with some feedback.

* removed needless "with PGroonga" from sentence (index.md)
* fixed folded long line (follow style with other sentences) (autocomplete.md)
* added prefix RK search (Romaji) example (autocomplete.md)
* fixed formatted SQL result (added prefix "--" for them) (autocomplete.md)

**Pull request merged**